src/components/map/layers/mapIcons/NodeMarker.tsx

```diff
@@ -193,9 +195,15 @@ class NodeMarker extends Component<INodeMarkerProps> {
     }
 
     render() {
-
         const icon = createDivIcon(
-                <div className={classnames(s.nodeBase, ...this.getMarkerClasses())}>
+                <div
+                    className={classnames(...this.getMarkerClasses())}
+                    style={{
+                        borderColor: this.props.color,
+                        backgroundColor: this.props.color,
+                    }}
```

Comment:
These style props weren't added to renderAdditionalLocations()'s createDivIcon functions.

I see why however, they that is not needed at the moment. How about adding a card to refactor renderAdditionalLocations out, so that NodeMarker would render only one <Marker />

Reply:
https://github.com/HSLdevcom/jore-map-ui/issues/797